Sources/NIOCore/EventLoop+Deprecated.swift

```diff
@@ -0,0 +1,29 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2021 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+extension EventLoop {
+    @inlinable
+    @_disfavoredOverload
```

Comment:
I don't think `_disfavoredOverload` should be necessary: the Swift ambiguity rules should prefer the "more specific" implementations, which don't have `file:line:` defaulted.